### PR TITLE
Sonic TT 16-Bit: Refactor port

### PR DESCRIPTION
**Pull request by JeodC**
- Do texture repacking and audio compression
- Only source from 64bit apk (no more 32bit)
- Use newer gmloadernext
- Fix controls for TSP

### CODE REVIEW

File: ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh [hunk 1]
```diff
@@ -13,100 +13,50 @@ else
...
-if [ -f "installed.64bit" ]; then
-    GMLOADER="gmloadernext.aarch64"
-    export LD_LIBRARY_PATH="/usr/libs:$GAMEDIR/libs:$LD_LIBRARY_PATH"
-elif [ -f "installed.32bit" ]; then
-    export PORT_32BIT="Y"
-    export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs32:$LD_LIBRARY_PATH"
-    GMLOADER="gmloadernext.armhf"
+# Display loading splash
+if [ -f "$GAMEDIR/patchlog.txt" ]; then
+    $ESUDO ./libs/splash "splash.png" 1
```
Comment (cdeletre) [inline ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh:52]:
```suggestion
[ "$CFW_NAME" == "muOS" ] && $ESUDO ./libs/splash "splash.png" 1 
```

File: ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh [hunk 2]
```diff
@@ -13,100 +13,50 @@ else
...
+    $ESUDO ./libs/splash "splash.png" 2000
 fi
 
 # Run the game
-echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-$GPTOKEYB "$GMLOADER" xbox360 &
-./$GMLOADER game.apk
+$GPTOKEYB "gmloadernext" -c "./sonic.gptk" &
+pm_platform_helper "$GAMEDIR/gmloadernext"
+./gmloadernext game.apk
```
Comment (cdeletre) [inline ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh:59]: Does your build use the `game.apk` argument ?

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — CHANGES_REQUESTED** [review]
see comments from @cdeletre

**Review by cdeletre — CHANGES_REQUESTED** [review]
See my request changes :)

File: ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh (hunk 2 above)
Comment (cdeletre) [reply ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh:59]: I've checked on your UFO 50 port and it indeed doesn't use it :)

Comment (cdeletre) [inline ports/sonictt16b/Sonic Triple Trouble - 16-Bit.sh:59]:
```suggestion
./gmloadernext
```

**Review by cdeletre — CHANGES_REQUESTED** [review]

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**